### PR TITLE
Change directory structure for storing dev keys (~/.near/netoworkid)

### PR DESCRIPTION
**Pull request by janedegtiareva**

### CODE REVIEW

**Comment by janedegtiareva** [comment]
tests pass locally

File: signing/unencrypted_file_system_keystore.js [hunk 1]
```diff
@@ -18,8 +17,11 @@ class UnencryptedFileSystemKeyStore {
      * @param {string} keypair 
      */
     async setKey(accountId, keypair) {
-        if (!await promisify(fs.exists)(keyDir)){
```
Comment (vgrichina) [inline signing/unencrypted_file_system_keystore.js]: nit: I think it makes sense to `promisify` all used `fs` functions at once after requiring `fs` vs doing it inline

**Comment by vgrichina** [comment]
Make sure you also do `npm run fix`, there is some eslint stuff.

**Comment by vgrichina** [comment]
@janedegtiareva closing this for now as got stale, feel free to re-open after rebasing / merging with master

**Pull request closed without merging**